packages/palette/src/elements/Select/Select.test.tsx

```diff
@@ -1,19 +1,31 @@
 import { mount } from "enzyme"
 import React from "react"
 import { SelectSmall } from "../Select"
+import { LargeSelect } from "./Select"
 
 describe("Select", () => {
+  const options = [
+    {
+      text: "First option",
+      value: "firstOption",
+    },
+    {
+      text: "Second option that is really long",
+      value: "secondOption",
+    },
+  ]
+  describe("LargeSelect", () => {
+    it("renders with a placeholder value", () => {
+      const wrapper = mount(
+        <LargeSelect options={options} placeholder="I'm a placeholder" />
```

Comment:
Why not set `placeholder` to just be `defaultValue`? We should be able to set it in this way anyways, and it will silence the warning from React.